packages/breadcrumb/src/js/breadcrumb.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+/**
+ * Breadcrumb module
+ * @file Handles interaction behaviour for the breadcrumb.
+ */
+
+class Breadcrumb {
+  constructor(el) {
+    this.init(el);
+  }
+
+  init(el) {
+    const breadcrumbs = el.querySelectorAll(".uq-breadcrumb__item");
+
+    // truncate if more than 4 options
+    if (breadcrumbs.length > 4) {
+      var crumbs = Array.from(breadcrumbs);
+
+      // create the expand crumb link
+      var expandCrumb = document.createElement("li");
+      expandCrumb.classList.add("uq-breadcrumb__item", "uq-breadcrumb__expand");
+      expandCrumb.innerHTML = '<a class="uq-breadcrumb__link">...</a>';
```

Comment:
I'd recommend moving the creation of objects into their own methods. These can then be accessed when necessary. A quick example:
```
/**
 * Create the expanded breadcrumb link.
 */
createBreadcrumbLink() {
  const expandCrumb = document.createElement("li");
  expandCrumb.classList.add("uq-breadcrumb__item", "uq-breadcrumb__expand");
  expandCrumb.innerHTML = '<a class="uq-breadcrumb__link">...</a>';
}
```